diesel/src/pg/connection/raw.rs

```diff
@@ -15,6 +15,12 @@ pub(super) struct RawConnection {
     internal_connection: NonNull<PGconn>,
 }
 
+impl From<&RawConnection> for *const PGconn {
+    fn from(raw_conn: &RawConnection) -> *const PGconn {
+        raw_conn.internal_connection.as_ptr()
+    }
+}
+
```

Comment:
I would prefer using the longer explicit variant here instead of having this impl in place.